DayZLifeServer/scripts/4_World/EventListener/DZLLoadOutListener.c

```diff
@@ -79,4 +92,23 @@ class DZLLoadOutListener: DZLBaseEventListener {
             }
         }
     }
+
+    private bool IsCooldownActive(PlayerBase player) {
+        string playerId = player.GetIdentity().GetId();
+        float currentTime = GetGame().GetTime() / 1000.0; // Get the current time in seconds
+        if (playerLoadoutAccessTimes.Contains(playerId)) {
```

Comment:
🔧 I am a fan of early exit.
 if (!playerLoadoutAccessTimes.Contains(playerId)) return false;